rxnetty/src/test/java/io/reactivex/netty/protocol/http/UnicastContentSubjectTest.java

```diff
@@ -149,6 +149,18 @@ public void call(ByteBuf byteBuf) {
         Assert.assertEquals("Byte buffer not released.", 0, last.refCnt());
     }
 
+    @Test
+    public void testByteBufReleaseWithTimeout() throws Exception {
+        UnicastContentSubject<ByteBuf> subject = UnicastContentSubject.create(100, TimeUnit.MILLISECONDS);
+        ByteBuf buffer = Unpooled.buffer();
+
+        subject.onNext(buffer);
+        subject.onCompleted();
+
+        Thread.sleep(500);
```

Comment:
This can be better done by passing a [`TestScheduler`](http://reactivex.io/RxJava/javadoc/rx/schedulers/TestScheduler.html) to `UnicastContentSubject.create()`